src/cdrizzleapi.c

```diff
@@ -610,9 +926,10 @@ clip_polygon_wrap(PyObject *self, PyObject *args) {
 
 static struct PyMethodDef cdrizzle_methods[] = {
     {"tdriz", (PyCFunction)tdriz, METH_VARARGS | METH_KEYWORDS,
-     "tdriz(image, weights, pixmap, output, counts, context, uniqid, xmin, "
-     "xmax, ymin, ymax, scale, pixfrac, kernel, in_units, expscale, wtscale, "
-     "fillstr)"},
+     "tdriz(image, weights, pixmap, output, counts, context, image2, "
+     "counts2, "
```

Comment:
I don't see counts2 anywhere else - is it supposed to be here?

Reply:
no, no need for `counts2`. Just square `counts`

Reply:
I think this one reference to counts2 still needs to be removed.  Other than that, all your changes look good to me.

Reply:
Done